src/gt4py/backend/gtc_backend/defir_to_gtir.py

```diff
@@ -17,22 +31,25 @@ class DefIRToGTIR(IRNodeVisitor):
     def apply(cls, root, **kwargs):
         return cls().visit(root)
 
-    GT4PY_ITERATIONORDER_TO_GTIR_LOOPORDER: ClassVar[Mapping[IterationOrder, common.LoopOrder]] = MappingProxyType(
+    GT4PY_ITERATIONORDER_TO_GTIR_LOOPORDER: ClassVar[
+        Mapping[IterationOrder, common.LoopOrder]
+    ] = MappingProxyType(
```

Comment:
I wonder if there is a reason this is not a simple `Dict`

Reply:
Does it just come from observing that `Enum`s use MappingProxyType? The way I have seen these X_TO_Y mappings used, a `Dict` would be fine too.

Reply:
Yes, there is nothing special about these mappings and they could be also dicts. I think MappingProxyTypes were used here just to protect new Python users to make weird bugs by accidentally modifying the mappings.